examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal

```diff
@@ -0,0 +1,61 @@
+import ballerina/config;
+import ballerina/http;
+import ballerina/log;
+
+// Define the basic auth client endpoint to call the backend services.
+// Basic authentication is enabled by setting the `scheme: http:BASIC_AUTH`
```

Comment:
This is description is quite same with secured_client_with_basic_auth.description. Shall we refactor?

Reply:
Fixed